media/js/client.js

```diff
@@ -297,6 +297,7 @@
                 id: user.id
             });
             target && target.set(user);
+            room.trigger("user:update", user);
```

Comment:
You should be able to go through a backbone model change event rather than triggering your own